content/en/localstack/persistence-mechanism/index.md

```diff
@@ -48,46 +49,21 @@ Otherwise, the endpoint will inform you that the mechanism is disabled.
-   Rather, diverging states is the more likely outcome.
-   This limitation is not the result of any implementation details related to LocalStack, but a result of how related AWS resources reference each other.
-   For instance, sending messages to an SQS queue is done by providing the queue URL.
-   However, there is no guarantee that our `sample-queue` will have the same URL after a reboot, which then would result in `NonExistingQueue` during the replay of `send-message` API calls.
-   For this reason, LocalStack provides no guarantees of the correctness and completeness for any state restored using the *record-and-replay* approach.
-3. The state recorded this way cannot be used to create [CloudPods]({{< ref "cloud-pods" >}}).
-
-### Persistence Mechanism - Pro Version
-
-The persistence mechanism of the Pro version is much more sophisticated and is based on *serialized state*.
+The persistence mechanism in LocalStack is a sophisticated approach based on *serialized state*.
 Starting the Pro version of LocalStack will traverse the `DATA_DIR` root folder recursively and directly deserialize the file into the application state.
 Typically, each service has one state file for each region.
 
-Each serialization mechanism has its own root folder.
-As of now, all supported services are serialized as pickle files, except for Kinesis (which is serialized as JSON) and DynamoDB (which is serialized as an SQLite database).
-This is illustrated with the diagram below.
+Each serialization mechanism has its root folder.
+As of now, all supported services are serialized as pickle files. 
+Particular services, in addition to their pickled files, can serialize additional artifacts.
+For instance, Kinesis persists some data in form of JSON while DynamoDB serializes a SQLite database.
+This is illustrated in the diagram below.
 
 ![Structure of the DATA_DIR](datadir_structure.png)
 
-This approach does not suffer from the same limitations as *record-and-replay*.
 Restoring the state -- even for large projects -- usually only takes a few milliseconds.
-Moreover, since the files store accurate snapshots of application state it can restore a state that is identical to the one prior to the reboot.
+Moreover, since the files store accurate snapshots of the application state they can restore a state that is identical to the before the reboot.
 
-However, the limitation of this approach is that it is not quite as generic.
-The *record-and-replay* approach can treat the AWS services as "black-boxes", i.e. the persistence mechanism is entirely decoupled from the underlying services, while the *serialized state* approach is coupled to the serialization mechanism of each service.
+The most notable limitation of this approach it's its generalizability.
+In fact, despite most of the supported AWS services serialize their state in a similar way, particular services might implement such a serialization in a completely different fashion.
```

Comment:
Wondering if we could simplify the last paragraph (last 3 lines) here a bit. I think users probably care most about which services have support for persistence (which should be almost all services at this stage..?), not so much about the limitations of the internal storage mechanism, or how it differs between services. 

We had this paragraph in here previously to compare the 2 persistence approaches, but probably we can simply remove lines 67-69? (and instead mention any services that do **not** yet support persistence, if there are any..)